### ISSUE TITLE
A11Y issue when using pixel unit

### ISSUE THREAD

**Issue by Nightapes** [angular-split/angular-split#323]
**Describe the bug**
When using as-split with unit `pixel` the gutters with role `slider` fail a11y tests. 
All mandatory aria attributes are not set like
```
aria-valuemin="40"
aria-valuenow="298"
aria-valuetext="298 pixel"
```
 
**To Reproduce**

https://angular-split.github.io/examples/nested-split with unit `pixel` and check html code if aria attributes are set

**Comment by SanderElias** [comment]
Closing this as it received no attention for too long.
Although this sometimes seems a real issue, a PR addressing this would always be welcome!
